recipes/aioshutil/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "aioshutil" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aioshutil-{{ version }}.tar.gz
+  sha256: ddabe1748c8a71ec3c7d213a4d0cf58fb495c71419334a0575b1f8a3be8373e8
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools-scm
+    - pip
+  run:
+    - python
+    - typing-extensions  # [py<310]
```

And unconditionally depend on typing-extensions:
```suggestion
requirements:
  host:
    - python >=3.8
    - setuptools-scm
    - pip
  run:
    - python >=3.8
    - typing-extensions
```